src/GithubFilesBundle/Service/GithubImport.php

```diff
@@ -165,10 +174,11 @@ public function saveAssociations($position, $content, $lsItemKeys, LsItem $lsIte
         $fieldsAndTypes = LsAssociation::allTypesForImportFromCSV();
         // We don't use is_child_of because that it alaready used to create parents relations before. :)
         // checking each association field
+        $sequenceNumber = $content[$position][trim($lsItemKeys['sequenceNumber'])];
         foreach ($fieldsAndTypes as $fieldName => $assocType){
```

Comment:
Due to `sequenceNumber` being added to the `$fieldsAndTypes` array, this will try to create an association of type `sequenceNumber` if it finds data.  This is probably not what is wanted.